src/instructlab/data/generate.py

```diff
@@ -34,11 +34,19 @@
     default=config.DEFAULT_CHUNK_WORD_COUNT,
     show_default=True,
 )
+# TODO - DEPRECATED - Remove in a future release
 @click.option(
     "--num-instructions",
     type=click.INT,
-    help="Number of instructions to generate.",
-    default=config.DEFAULT_NUM_INSTRUCTIONS,
+    help="DEPRECATED. See --sdg-scale-factor",
+    default=-1,
+    hidden=True,
```

Comment:
deprecated
```suggestion
    hidden=True,
    deprecated=True,
```

Reply:
I tried to look through the click docs to see if this was a thing and didn't see it. I found it for commands but not options. No surprise that the docs aren't perfect, or my reading isn't perfect. :) I'll add this when I update the PR

Reply:
added

Reply:
@leseb so i wasn't crazy. this isn't supported

```
TypeError: Parameter.__init__() got an unexpected keyword argument 'deprecated'
```